src/gui/MainPane/RoomList.qml

```diff
@@ -73,6 +73,48 @@ HListView {
             keepListCentered = true
     }
 
+    function indexOfRoomId(roomId) {
+        for (let i = 0; i < model.count; i++) {
+            const item = model.get(i)
+            if (item.type === "Room" && item.id === roomId) {
+                return i
+            }
+        }
+        return -1
+
+    }
+
+    // Description can be either room id
+    // or space-separated account id and room id.
+    // If only room id, the first account with this room is used.
+    function showRoomByDescription(description) {
+        const spaceIndex = description.indexOf(" ")
```

Comment:
You can do `const [roomId, accountId] = description.split(" ").reverse()`, that's more explicit and gives you `accountId` and `roomId` variables directly